protocols/messages.go

```diff
@@ -0,0 +1,50 @@
+package protocols
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/statechannels/go-nitro/channel/state"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// Message is an object to be sent across the wire. It can contain a proposal and signed state hashes, and is addressed to a counterparty.
```

Comment:
```suggestion
// Message is an object to be sent across the wire. It can contain a proposal and signed states, and is addressed to a counterparty.
```

Reply:
f87b36c